packages/core/upload/admin/src/pages/App/ConfigureTheView/tests/ConfigureTheView.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { render as renderTL, screen, waitFor } from '@testing-library/react';
+import { TrackingProvider } from '@strapi/helper-plugin';
+import { MemoryRouter } from 'react-router-dom';
+import { IntlProvider } from 'react-intl';
+
+import { ConfigureTheView } from '..';
+
+jest.mock('../../../../hooks/useConfig');
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  useNotification: jest.fn(() => jest.fn()),
+}));
+
+const renderConfigure = (
+  config = {
+    data: {
+      pageSize: '10',
+    },
+  }
+) =>
+  renderTL(
+    <IntlProvider locale="en" messages={{}}>
+      <TrackingProvider>
+        <ThemeProvider theme={lightTheme}>
+          <MemoryRouter>
+            <ConfigureTheView config={config} />
+          </MemoryRouter>
+        </ThemeProvider>
+      </TrackingProvider>
+    </IntlProvider>
+  );
+
+describe('Upload - Configure', () => {
```

Comment:
Is it possible to test some of the user actions here? Updating a setting, for example.